examples/basic/android/app/build.gradle

```diff
@@ -147,6 +155,8 @@ dependencies {
             because("kotlin-stdlib-jdk8 is now a part of kotlin-stdlib")
         }
     }
+    // coreLibraryDesugaring is mandatory to be able to build exoplayer from source
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.4'
```

Comment:
Also this

Reply:
The same :) 

Yes you are right, but this is already in this section:
### Desugaring
I pointed the google documentation instead of copying it !